.golangci.yaml

```diff
@@ -1,209 +1,235 @@
   go: "1.24"
 linters:
-  disable-all: true
+  default: none
   enable:
-  - asasalint
-  - asciicheck
-  - bidichk
-  - bodyclose
-  - containedctx
-  - copyloopvar
-  - decorder
-  - dogsled
-  - dupl
-  - dupword
-  - durationcheck
-  - errcheck
-  - errchkjson
-  - errname
-  - errorlint
-  - exhaustive
-  - exptostd
-  - fatcontext
-  - forbidigo
-  - forcetypeassert
-  - gci
-  - ginkgolinter
-  - gocheckcompilerdirectives
-  - gochecksumtype
-  - goconst
-  - gocritic
-  - godot
-  - gofmt
-  - goimports
-  - goprintffuncname
-  - gosec
-  - gosimple
-  - gosmopolitan
-  - govet
-  - iface
-  - importas
-  - ineffassign
-  - intrange
-  - loggercheck
-  - makezero
-  - mirror
-  - misspell
-  - mnd
-  - nakedret
-  - nilerr
-  - nilnesserr
-  - nilnil
-  - noctx
-  - nolintlint
-  - nosprintfhostport
-  - perfsprint
-  - prealloc
-  - predeclared
-  - reassign
-  - revive
-  - rowserrcheck
-  - staticcheck
-  - stylecheck
-  - tagliatelle
-  - testifylint
-  - thelper
-  - tparallel
-  - typecheck
-  - unconvert
-  - unparam
-  - unused
-  - usestdlibvars
-  - usetesting
-  - whitespace
-  # Run with --fast=false for more extensive checks
-  fast: true
-
-linters-settings:
-  gosec:
-    severity: medium
-    confidence: medium
-    concurrency: 8
-  importas:
-    no-unaliased: true
-    alias:
-    # Kubernetes
-    - pkg: k8s.io/api/core/v1
-      alias: corev1
-    - pkg: k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1
-      alias: apiextensionsv1
-    - pkg: k8s.io/apimachinery/pkg/apis/meta/v1
-      alias: metav1
-    - pkg: k8s.io/apimachinery/pkg/api/errors
-      alias: apierrors
-    # Controller Runtime
-    - pkg: sigs.k8s.io/controller-runtime
-      alias: ctrl
-    - pkg: sigs.k8s.io/cluster-api/api/v1alpha4
-      alias: clusterv1alpha4
-    - pkg: sigs.k8s.io/cluster-api/api/v1beta2
-      alias: clusterv1
-    - pkg: sigs.k8s.io/cluster-api/api/core/v1beta1
-      alias: clusterv1beta1
-    - pkg: sigs.k8s.io/cluster-api/api/ipam/v1beta1
-      alias: capipamv1beta1
-    # IPAM
-    - pkg: github.com/metal3-io/ip-address-manager/api/v1alpha1
-      alias: ipamv1
-  nolintlint:
-    allow-unused: false
-    require-specific: true
-  gocritic:
-    enabled-tags:
-    - experimental
-    disabled-checks:
-    - appendAssign
-    - dupImport # https://github.com/go-critic/go-critic/issues/845
-    - evalOrder
-    - ifElseChain
-    - octalLiteral
-    - regexpSimplify
-    - sloppyReassign
-    - truncateCmp
-    - typeDefFirst
-    - unnamedResult
-    - unnecessaryDefer
-    - whyNoLint
-    - wrapperFunc
-  ginkgolinter:
-    forbid-focus-container: true
-    force-expect-to: true
-  tagliatelle:
-    case:
-      rules:
-        json: goCamel
-  govet:
-    enable:
-    - shadow
-issues:
-  exclude-dirs:
-  - mock*
-  exclude-files:
-  - "zz_generated.*\\.go$"
-  - ".*conversion.*\\.go$"
-  exclude-rules:
-  - path: _test\.go
-    linters:
-    - unused
+    - asasalint
```

Comment:
Please don't change the indentation. We follow k8s identation, ie. no leading spaces for lists.
I know the migrate tool did this for you, but change them back to cut down the diff.